src/client/app/profile/patient-profile.component.ts

```diff
@@ -77,13 +80,40 @@ export class PatientProfileComponent implements OnInit {
                     location: [selectedLocation],
                     address: null,
                     description: this.user.description,
-                    documentType:null,
+                    documentType: null,
                     documentTitle: null,
                     documentDescription: null
                 });
             });
     }
 
+    getAllergies() {
+        this.sharedService.getAllergies()
+            .subscribe(allergies => {
+                allergies.map((allergy: any) => {
```

Comment:
Similarly in this component as well. Check the type

Reply:
allergy also is a JSON object so it cant be of string type.

Reply:
Alright